sections/header.liquid

```diff
@@ -89,6 +89,7 @@
   }
 
   .section-header {
+    position: sticky; /* This is for fixing a Safari z-index issue. PR #2147 */
```

Comment:
> I was applying too specifically to elements vs the general header container

^ exactly. Got me too. But because Eugene found that putting the position sticky there worked, it got me trying this on the same element.